docs/source/usage/lease_notification.rst

```diff
@@ -0,0 +1,30 @@
+Lease Notification
```

Comment:
This should probably be renamed; I'd call it something like 'Automated Workflows', since the notifications by themselves don't do much.

Reply:
I've changed the title to 'Automated Workflows'